genappbuilder/src/main/java/genappbuilder/v1beta/Search.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package genappbuilder.v1beta;
+
+// [START genappbuilder_search]
```

Comment:
Region tag prefix must match the product/API name. Please use `discoveryengine_search`
Update the END tag as well 
Also update all place where the region tag is used

Reply:
This is a special case... Generative AI App Builder is using the Discovery Engine API and it has a distinct region tag. https://devrel.corp.google.com/admin/products/649